pytensor/tensor/subtensor.py

```diff
@@ -1504,16 +1508,18 @@ def inc_subtensor(
     --------
     To replicate the expression ``r[10:] += 5``:
 
-    ..code-block:: python
+    .. code-block:: python
+
+        from pytensor.tensor import ivector, inc_subtensor
 
-        r = ivector()
+        r = ivector("r")
         new_r = inc_subtensor(r[10:], 5)
 
     To replicate the expression ``r[[0, 1, 0]] += 5``:
 
-    ..code-block:: python
+    .. code-block:: python
 
-        r = ivector()
+        r = ivector("r")
         new_r = inc_subtensor(r[10:], 5, ignore_duplicates=True)
```

Comment:
Not sure if the `ignore_duplicates` are forwarded though.
```suggestion
        new_r = r[10:].inc(5, ignore_duplicates=True)
```